packages/vue/src/pinia.ts

```diff
@@ -0,0 +1,87 @@
+import { addBreadcrumb, getClient, getCurrentScope, getGlobalScope } from '@sentry/core';
+import { addNonEnumerableProperty } from '@sentry/utils';
+
+// Inline PiniaPlugin type
+type PiniaPlugin = (context: {
+  store: {
+    $id: string;
+    $state: unknown;
+    $onAction: (callback: (context: { name: string; after: (callback: () => void) => void }) => void) => void;
+  };
+}) => void;
+
+type SentryPiniaPluginOptions = {
+  attachPiniaState?: boolean;
+  actionTransformer: (action: any) => any;
+  stateTransformer: (state: any) => any;
```

Comment:
I think it would be nice to have tests for those :)

Reply:
Added 👍  https://github.com/getsentry/sentry-javascript/pull/13841/commits/c7cfa72a549213f1ab0b8bae156b83c972f8db43